### PR TITLE
docs(0.12.x/admin-api) document /upstream/:id/health endpoint

### PR DESCRIPTION
**Pull request by hishamhm**

### CODE REVIEW

File: app/docs/0.12.x/admin-api.md [hunk 1]
```diff
@@ -2100,7 +2173,7 @@ target](#delete-target) instead.
 
 #### Endpoint
 
-<div class="endpoint post">/upstreams/{upstream name or id}/targets/{target or id}/healthy</div>
+<div class="endpoint post">/upstreams/{upstream name or id}/targets/{target or id}/unhealthy</div>
```
Comment (thibaultcha) [inline app/docs/0.12.x/admin-api.md]: oops :/ I think this also belongs to master in the 0.12 version of the docs? So we can deploy it asap? (I know you opened the other PR against master originally)

Comment (hishamhm) [reply app/docs/0.12.x/admin-api.md]: PR to master: https://github.com/Kong/getkong.org/pull/604

**Comment by hishamhm** [comment]
Updated with uppercase entities.

**Comment by thibaultcha** [comment]
Merged with typo fix

**Pull request closed without merging**